Makefile

```diff
@@ -42,24 +43,23 @@ routes:
 
 .PHONY: test
 test:
-	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER}  pytest --cov=app tests/
+	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER} pytest --cov=app tests/
 
 .PHONY: lint
 lint:
-	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER}  flake8 src/app --statistics --count
+	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER} flake8 src/app --statistics --count
 
 .PHONY: help
 help: build
-	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER}  flask --help
+	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER} flask --help
 
 
 .PHONY: build
 build:
-	${DOCKER_COMPOSE} build
+	${DOCKER_COMPOSE} build --pull
 
 .PHONY: setup
 setup: build
-	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER}  flask db-migrate create-tables
-	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER}  flask db stamp head
-	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER}  flask db-migrate init
-
+	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER} -m flask db-migrate create-tables
```

Comment:
For some reason I think that this doesn't work since it will call
`docker-compose run resources01 -m flask db stamp head`
This is missing `python` or needs to remove the `-m`?

Reply:
There are two spaces after `${RESOURCES_CONTAINER}` in the previous commit.